lib/api/workloadendpoint.go

```diff
@@ -36,13 +36,17 @@ type WorkloadEndpointMetadata struct {
 	Name string `json:"name,omitempty" validate:"omitempty,namespacedname"`
 
 	// The name of the workload.
-	Workload string `json:"workload,omitempty" valid:"omitempty,namespacedname"`
+	Workload string `json:"workload,omitempty" validate:"omitempty,namespacedname"`
+
+	// ActiveInstanceID contains an identifier for the container networked for
```

Comment:
Can we move this to be above the Label field.

I think the comment should:  Only mention the word container by way of an example, mention that it is not a unique identifier for the endpoint rather it is additional instance data that may be used by the orchestrator to provide additional consistency checks.  For example...